tests/std/tests/P0218R1_filesystem/test.cpp

```diff
@@ -3608,6 +3609,17 @@ void test_create_directory() {
         }
     }
 
+    // test empty path

```

Comment:
A few general comments:
1. This is in the middle of function `test_create_directory` (which primarily tests `std::fs::create_directory`), I think the following function `test_create_dirs_and_remove_all` (which primarily tests `std::fs::create_directories` and `std::fs::remove_all`) would be a better place.
1. Since we're adding this test case in response to a bug report, we'd conventionally include the bug# and a brief description in the comment (as on line 3623 below). I'd expect to see something about GH-1283 here.
1. Using the `EXPECT` machinery would better integrate this test case with the rest of the test, which also generally tests both throwing and non-throwing paths. Something like:
```c++
EXPECT(create_directories(path{}, ec) == false);
EXPECT(bad(ec));
EXPECT(throws_filesystem_error([] { create_directories(path{}); }, "create_directories", path{});
```
should do the job.

Reply:
I'm sorry. It seems I was too careless about these tests.